scripts/train_agents.py

```diff
@@ -83,25 +83,11 @@ def SP_w_SP_Types(args,
 
 
 def FCP(args, pop_force_training, fcp_force_training, parallel):
```

Comment:
So there are two types of FCP, one is the traditional FCP that uses random teammates (i.e. ALL_MIX), one is our own version that uses certain types HIGH_FIRST, MEDIUM_FIRST, etc. The reason we have our version is that when we used the traditional FCP it got ~0 reward and then we decided to add all these different types for teammates_collection.

I suggest creating two functions: `FCP_traditional` that uses `ALL_MIX`, `FCP_mhri` that uses the default setting we have right now in the main branch.

Reply:
added!